src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -7,19 +7,46 @@
 ///   .my-app {
 ///     @include mat-mdc-strong-focus-indicators($config);
 ///   }
+
+// Mixin that handles null values for CSS variables
+@mixin _create-variable($name, $value) {
```

Comment:
@crisbeto Do we want different legacy & MDC `strong-focus-indicators` mixins? As they currently stand, the mixins look 100% (?) identical. I think it makes sense to keep compatibility with the current two mixins (for now). I think Pantheon might run into issues if we provide just a single combined legacy/MDC mixin. If we do have different mixins, I think we'll either (1) need different CSS variables for each or (2) need to scope them within the `.mat-focus-indicator` and `.mat-mdc-focus-indicator` selectors.

Even if we have different mixins, given they share so much underlying logic it probably makes sense to pull out shared logic into a common helper somewhere (especially extremely common helpers like `_create-variable`). Where might be a good place for such a helper to live?

Reply:
> As they currently stand, the mixins look 100% (?) identical

That's why I was pushing to combine the classes initially. If we're keeping them separate, we might as well have separate mixins too. We can still pull the common code out into a private mixin that accepts a prefix for the variable name.

Reply:
And thoughts on:

If we do have different mixins, I think we'll either (1) need different CSS variables for each or (2) need to scope them within the `.mat-focus-indicator` and `.mat-mdc-focus-indicator` selectors.

Reply:
According to Kristiyan, since the logic is similar, we might be able to combine the two strong-focus-indicator mixins into one private mixin along with the helper mixins and move them to a file under the src/material folder.

Reply:
@zelliott I think that we should have two different sets of variables.